src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java

```diff
@@ -172,7 +176,18 @@ protected boolean submit(O op) throws InterruptRunException, IllegalStateExcepti
 						// if(success) return true;
 						break;
 					case READ:
-						// if(success) return true;
+						final String path = pathOp.dstPath();
+						final String scopeName = path.substring(0, path.indexOf("/"));
+						final String streamName = path.substring(path.indexOf("/") + 1);
+						if ((scopeMap.get(scopeName) == null) || (scopeMap.get(scopeName).get(streamName) == null)) {
```

Comment:
@akurilov I would have done it if we had to handle these situations differently. I don't quite see why we need nesting if both conditions are handled in the same way. Though i will fix it for sure, i would also be happy if you give me a reason.

Reply:
I think we should to replace IF condition in 182 line on 
`if ((null != scopeMap.get(scopeName) && null != scopeMap.get(scopeName).get(streamName)))`
Because now case with NULL.get(streamName) doesn't handle at all.
I also think that nested if here is unnecessary.